### PR TITLE
refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

replace module from `ovh-angular-mondial-relay` to `ngOvhMondialRelay`
BREAKING CHANGE: module is now named `ngOvhMondialRelay`

2c7af22 — refactor: update stack with component-rollup-config

### Todo

* [ ] rename repository to `ng-ovh-mondial-relay`;

### Related PR

* [x] https://github.com/ovh-ux/component-rollup-config/pull/56

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  "plugins": [
+    "markdown"
+  ],
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "import/no-unresolved": 0
+  },
+  "overrides": [{
```
Comment (antleblanc) [inline .eslintrc.json]: According to the `README.md` file, these `overrides` aren't needed.

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,2 @@
 node_modules
-.work
-.idea
-bower_components
-npm-debug.log
-*.orig
-tools/keys
-src/*/*/Messages_*_*.json
+dist
```
Comment (antleblanc) [inline .gitignore:2]: +`tools`

File: .lintstagedrc.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "*.js": [
```
Comment (antleblanc) [inline .lintstagedrc.json:5]: missing both `*.less` and `*.html`

File: README.md [hunk 4]
```diff
@@ -1,175 +1,37 @@
...
-The more convinent is to convert these files in json and provide partial translation files via `mondialRelayProvider`
+## Contributing
 
-## Requests to API
-You need 2 services to request API:
-* `SupplyMondialRelay` ([https://api.ovh.com/1.0/supply/mondialRelay](https://api.ovh.com/console/#/supply/mondialRelay#POST))
-* `User` ([https://api.ovh.com/1.0/me](https://api.ovh.com/console/#/me#GET))
-If not provided, these 2 services will be injected from OvhApiServices so be sure to either provide the services or include OvhApiServices
-as a dependency of your project (see: [https://github.com/ovh-ux/ovh-api-services](https://github.com/ovh-ux/ovh-api-services)).
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ng-ovh-simple-country-list/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ng-ovh-simple-country-list/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ng-ovh-mondial-relay/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ng-ovh-mondial-relay/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+import image from 'rollup-plugin-img';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const esConfig = config.es();
+esConfig.plugins.push(image());
```
Comment (antleblanc) [inline rollup.config.js]: Shouldn't be handled directly through shared configuration `components-rollup-config` itself?

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: rollup.config.js (hunk 5 above)
Comment (cbourgois) [reply rollup.config.js]: Make sense.

Comment (cbourgois) [reply rollup.config.js]: Moved in https://github.com/ovh-ux/component-rollup-config/pull/56

File: package.json [hunk 6]
```diff
@@ -1,45 +1,62 @@
...
-    "karma": "~0.13.0",
-    "karma-jasmine": "~0.3.0",
-    "karma-ng-html2js-preprocessor": "~0.1.0",
-    "karma-phantomjs-launcher": "~1.0.0",
-    "karma-spec-reporter": "0.0.26",
-    "matchdep": "^0.3.0",
-    "phantomjs-prebuilt": "^2.1.7"
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@ovh-ux/component-rollup-config": "ovh-ux/component-rollup-config#feat/image",
```
Comment (antleblanc) [inline package.json]: https://github.com/ovh-ux/component-rollup-config/blob/master/CHANGELOG.md#500-beta9-2019-02-07

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: rollup.config.js (hunk 5 above)
Comment (cbourgois) [reply rollup.config.js]: `ovh-ux/component-rollup-config` bumped to `5.0.0-beta.9`

File: package.json (hunk 6 above)
Comment (cbourgois) [reply package.json]: `ovh-ux/component-rollup-config` bumped to `5.0.0-beta.9`

**Review by antleblanc — APPROVED** [review]

**Pull request merged**